### ISSUE TITLE
The number of iterations and the steps to drop learning rate?

### ISSUE THREAD

**Issue by PHDJieFu** [tobyperrett/trx#6]
Thanks for your published work! 
When I reproduce your results on the Kinetics-100 dataset (with a total of 100k iterations), I find that the accuracy of the model will drop later. When will the performance of the model reach the best in the training process？
In addition, can you give me more details about the number of training iterations and the steps to drop the learning rate on both SSV2 and Kinetics-100 datasets.  Thank you again for your help！

**Comment by tobyperrett** [comment]
Hi.  You'd use the validation set to determine when to stop training ( https://github.com/tobyperrett/trx/issues/3 ). I experimented with dropping the learning rate, but it didn't make much of a difference (<1%), so wasn't worth the extra computation time/hyperparameter search.
